versions/development/SPEC.md

```diff
@@ -344,20 +345,79 @@ Tasks and workflow inputs may be passed in from an external source, or they may
 
-| Escape Sequence | Meaning | \x Equivalent | Context |
-|----|---|------|--|
-|`\\`|`\`|`\x5C`||
-|`\n`|newline|`\x0A`||
-|`\t`|tab|`\x09`||
-|`\'`|single quote|`\x27`|within a single-quoted string|
-|`\"`|double quote|`\x22`|within a double-quoted string|
+| Escape Sequence | Meaning      | \x Equivalent | Context                       |
+| --------------- | ------------ | ------------- | ----------------------------- |
+| `\\`            | `\`          | `\x5C`        |                               |
+| `\n`            | newline      | `\x0A`        |                               |
+| `\t`            | tab          | `\x09`        |                               |
+| `\'`            | single quote | `\x27`        | within a single-quoted string |
+| `\"`            | double quote | `\x22`        | within a double-quoted string |
 
 Strings can also contain the following types of escape sequences:
 
 * An octal escape code starts with `\`, followed by 3 digits of value 0 through 7 inclusive.
 * A hexadecimal escape code starts with `\x`, followed by 2 hexadecimal digits `0-9a-fA-F`. 
 * A unicode code point starts with `\u` followed by 4 hexadecimal characters or `\U` followed by 8 hexadecimal characters `0-9a-fA-F`.
 
+##### Multi-line Strings
+
+Strings that begin and end with three consecutive single- or double-quotes may span multiple lines. The opening quotes of a multi-line string may be followed by whitespace (optional) and must have a newline before the first non-whitespace character; these leading whitespace/newline characters are removed. All subsequent non-empty lines are then used to determine the multi-line string's *common leading whitespace* - the minimum number of whitespace characters occuring before the first non-whitespace character in a line or the end of the line, whichever comes first. This common leading whitespace is stripped from the beginning of all the lines in the multi-line string.
+
+```wdl
+# These three strings are equivalent. The middle line of each is empty and so does not count
+# towards the common leading whitespace determination.
+
+String multi_line_A = '''
+        this is a
+
+          multi-line string'''
+
+String multi_line_B = """
+  this is a
+
+    multi-line string"""
+
+String multi_line_C = """
+this is a
+
+  multi-line string"""
+```
+String multi_line_D = """
+    this is a
+  a multi-line string"""
+Newline characters are not stripped out unless they are escaped, i.e. when the last character of a line is `\`.
+
+```wdl
+# These two strings are equivalent:
+String single_line = "this is a double-quoted string that contains no newlines"
+String multi_line_escaped = """
+  this is a \
+  double-quoted string \
+  that contains no newlines"""
+```
+
+Keep in mind that if the ending quotes are on a line by themselves and are preceeded by whitespace, that line is included when determining the common leading whitespace.
+
+```wdl
+# The following two strings are equivalent:
```

Comment:
Is one of the "two strings" missing?

Reply:
No, but maybe that's not clear. Please let me know if the change clears things up.